tests/unit/Claim/ClaimsTest.php

```diff
@@ -242,109 +229,109 @@ public function testAppendWithNonClaimFails() {
-		$this->assertEquals( $claim2, $claims->getClaimWithGuid( $claim2UpperGuid ) );
+		$this->assertEquals( $statement1, $claims->getClaimWithGuid( $claim1LowerGuid ) );
+		$this->assertEquals( $statement2, $claims->getClaimWithGuid( $claim2UpperGuid ) );
 	}
 
 	/**
 	 * Attempts to add Claims with no GUID set will fail.
 	 */
 	public function testNoGuidFailure() {
-		$claim = new Claim( new PropertyNoValueSnak( 42 ) );
+		$statement = new Statement( new PropertyNoValueSnak( 42 ) );
 		$list = new Claims();
 
 		$this->setExpectedException( 'InvalidArgumentException' );
-		$list->addClaim( $claim );
+		$list->addClaim( $statement );
 	}
 
 	public function testDuplicateClaims() {
-		$firstClaim = $this->makeClaim( new PropertyNoValueSnak( 42 ) );
-		$secondClaim = $this->makeClaim( new PropertyNoValueSnak( 42 ) );
+		$statement1 = $this->makeStatement( new PropertyNoValueSnak( 42 ) );
+		$statement2 = $this->makeStatement( new PropertyNoValueSnak( 42 ) );
 
 		$list = new Claims();
-		$list->addClaim( $firstClaim );
-		$list->addClaim( $secondClaim );
+		$list->addClaim( $statement1 );
+		$list->addClaim( $statement2 );
 
 		$this->assertEquals( 2, $list->count(), 'Adding two duplicates to an empty list should result in a count of two' );
 
-		$this->assertEquals( $firstClaim, $list->getClaimWithGuid( $firstClaim->getGuid() ) );
-		$this->assertEquals( $secondClaim, $list->getClaimWithGuid( $secondClaim->getGuid() ) );
+		$this->assertEquals( $statement1, $list->getClaimWithGuid( $statement1->getGuid() ) );
+		$this->assertEquals( $statement2, $list->getClaimWithGuid( $statement2->getGuid() ) );
 
-		$list->removeClaimWithGuid( $secondClaim->getGuid() );
+		$list->removeClaimWithGuid( $statement2->getGuid() );
 
-		$this->assertNotNull( $list->getClaimWithGuid( $firstClaim->getGuid() ) );
-		$this->assertNull( $list->getClaimWithGuid( $secondClaim->getGuid() ) );
+		$this->assertNotNull( $list->getClaimWithGuid( $statement1->getGuid() ) );
+		$this->assertNull( $list->getClaimWithGuid( $statement2->getGuid() ) );
 	}
 
 	public function testIterator() {
 		$expected = array(
-			'TESTCLAIM1' => $this->makeClaim( new PropertyNoValueSnak( new PropertyId( 'P42' ) ), 'testclaim1' ),
-			'TESTCLAIM2' => $this->makeClaim( new PropertySomeValueSnak( new PropertyId( 'P42' ) ), 'testclaim2' ),
-			'TESTCLAIM3' => $this->makeClaim( new PropertyNoValueSnak( new PropertyId( 'P23' ) ), 'testclaim3' ),
-			'TESTCLAIM4' => $this->makeClaim( new PropertyNoValueSnak( new PropertyId( 'P9000' ) ), 'testclaim4' ),
+			'TESTCLAIM1' => $this->makeStatement( new PropertyNoValueSnak( new PropertyId( 'P42' ) ), 'testclaim1' ),
+			'TESTCLAIM2' => $this->makeStatement( new PropertySomeValueSnak( new PropertyId( 'P42' ) ), 'testclaim2' ),
+			'TESTCLAIM3' => $this->makeStatement( new PropertyNoValueSnak( new PropertyId( 'P23' ) ), 'testclaim3' ),
+			'TESTCLAIM4' => $this->makeStatement( new PropertyNoValueSnak( new PropertyId( 'P9000' ) ), 'testclaim4' ),
```

Comment:
No. This is part of the Claims implementation (see `Claims::getGuidKey`), which did not changed.